build-binutils.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
+# Description: Builds a standalone copy of binutils
+
+import argparse
+import multiprocessing
+import os
+import pathlib
+import platform
+import shutil
+import subprocess
+import utils
+
+
+def host_arch_target():
+    """
```

Comment:
The other files uses `'`, this one seems to use `"`. I would choose one :-) My preference is `'` if a script language supports it (because many important text formats use `"`), but I don't know what the formatter does if you are using that one.

Reply:
I am sure it handles double quotes :) What I meant to ask is whether the formatter enforced a quote style in all cases (i.e. not just docstrings), i.e. if it rewrites them or just leaves whatever it finds.

Reply:
Oh. YAPF won't change the quoting style. Google's internal tool prefers double quotes for docstrings.